mobilizon_reshare/web/backend/main.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from fastapi import FastAPI
+from tortoise.contrib.pydantic import pydantic_model_creator
+
+from mobilizon_reshare.models.event import Event
+from mobilizon_reshare.storage.db import init as init_db, get_db_url
+
+app = FastAPI()
+event_pydantic = pydantic_model_creator(Event)
+
+
+logger = logging.getLogger(__name__)
+
+
+def check_database():
+    url = get_db_url()
+    if url.scheme == "sqlite":
+        logger.warning(
+            "Database is SQLite. This might create issues when running the web application. Please use a "
```

Comment:
Is this about concurrency?

Reply:
kinda. SQLite is not fit to serve concurrent requests on a web server.